src/main/java/app/linguistai/bmvp/controller/ProfileController.java

```diff
@@ -39,4 +44,10 @@ public ResponseEntity<Object> getProfile(@RequestHeader(Header.USER_EMAIL) Strin
         return Response.create("OK", HttpStatus.OK, userProfile);        
     }
 
+    @GetMapping(consumes = MediaType.APPLICATION_JSON_VALUE, path = "search")
+    public ResponseEntity<Object> searchUser(@Valid @RequestBody QUserSearch userSearch,
```

Comment:
Please add swagger annotation for controllers

Reply:
@elif-krvan will look at it later, I am merging this.